Servers/Administration/OJS.Servers.Administration/wwwroot/js/mvc-grid-form.js

```diff
@@ -1,37 +1,50 @@
-$(document).ready(function () {
-    $(":checkbox").on('click', function (event) {
-        if (event.target.value === "True") {
-            event.target.value = "False"
+$(document).ready(() => {
+    $(':checkbox').on('click', (event) => {
+        if (event.target.value === 'True') {
+            event.target.value = 'False';
         } else {
-            event.target.value = "True"
+            event.target.value = 'True';
         }
     });
 
     const forms = $('form');
 
     forms.find('select').select2();
 
-    forms.submit(function (ev) {
+    forms.submit((ev) => {
         const form = $(ev.target);
 
         form.find('select').prop('disabled', false);
 
-        form.find('fieldset').each(function (ev, fs) {
+        form.find('fieldset').each((ev, fs) => {
             const selectedCheckboxes = $(fs).find('input[type=checkbox]').toArray();
 
-            const getExpandElement = (checkbox) => {
-                const expandElement = $('#' + $(checkbox).attr("expand") + ' :input')[0];
+            const getExpandElements = (checkbox) => {
+                const expandElementId = $(checkbox).attr('expand');
+                const result = [];
 
-                return expandElement && expandElement.name && expandElement.value
-                    ? {name: expandElement.name, value: expandElement.value}
+                if (expandElementId) {
+                    const expandElements = $(`#${$(checkbox).attr('expand')} :input`);
```

Comment:
`$(checkbox).attr('expand')` is repeated - can you reuse it?

Reply:
Done.